spine/device_local.go

```diff
@@ -23,10 +23,16 @@ type DeviceLocalImpl struct {
 	serialNumber string
 }
 
-func NewDeviceLocalImpl(brandName, deviceModel, deviceCode, serialNumber, deviceAddress string, deviceType model.DeviceTypeType) *DeviceLocalImpl {
+func NewDeviceLocalImpl(brandName, deviceModel, deviceCode, serialNumber, deviceAddress string, deviceType model.DeviceTypeType, featureSet model.NetworkManagementFeatureSetType) *DeviceLocalImpl {
 	address := model.AddressDeviceType(deviceAddress)
+
+	var fSet *model.NetworkManagementFeatureSetType
+	if len(featureSet) != 0 {
```

Comment:
Was genau macht denn diese Codezeile? Ich dachte `len` wird für Arrays oder Maps verwendet. 
Welcher Wert muss denn als Argument übergeben werden, damit `len(featureSet) == 0` ist?

Reply:
Das ist die Länge des strings

Reply:
Aber wenn der Typ `model.NetworkManagementFeatureSetType` ist, kann er doch nie `== 0` sein.

Reply:
`model.NetworkManagementFeatureSetType` ist ein String, siehe https://github.com/DerAndereAndi/eebus-go/blob/dev/spine/model/networkmanagement.go#L19
Wenn ich in der `serviceDescription` keinen Wert dafür anlege, dann ist der leer. Kannst mal die HEMS oder EVSE Test app im Repo hier ausführen und da nen breakpoint setzen.

Das können wir erst besser machen, wenn `ServiceDescription` defaults bekommt und die gesetzten Werte auch prüft. Das haben wir noch nicht.